Updating "unified audit logs" to Microsoft 365 Audit (#6448)
MicrosoftDocs/microsoft-365-docs: microsoft-365/compliance/communication-compliance-siem.md

[Communication compliance](communication-compliance.md) is an insider risk solution in Microsoft 365 that helps minimize communication risks by helping you detect, capture, and act on inappropriate messages in your organization. Security information and event management (SIEM) solutions such as [Azure Sentinel](https://azure.microsoft.com/services/azure-sentinel) or [Splunk](https://www.splunk.com/) are commonly used to aggregate and track threats within an organization.

> A common need for organizations is to integrate communication compliance alerts and these SIEM solutions. With this integration, organizations can view communication compliance alerts in their SIEM solution and then remediate alerts within the communication compliance workflow and user experience. For example, an employee sends an offensive message to another employee and that message is detected by a communication compliance policy monitoring for offensive language. These events are tracked in the Microsoft 365 Audit (also known as "Unified Audit Log") by the communication compliance solution and imported in the SIEM solution. An alert is then triggered in the SIEM solution for the organization from events monitored in the Microsoft 365 Audit that are associated with communication compliance alerts. Investigators are notified of the alert in the SIEM solutions and then they investigate and remediate alert in the communication compliance solution.

Reviewer: ```suggestion
A common need for organizations is to integrate communication compliance alerts and these SIEM solutions. With this integration, organizations can view communication compliance alerts in their SIEM solution and then remediate alerts within the communication compliance workflow and user experience. For example, an employee sends an offensive message to another employee and that message is detected by a communication compliance policy monitoring for offensive language. These events are tracked in the Microsoft 365 Audit (also known as "unified audit log") by the communication compliance solution and imported into the SIEM solution. An alert is then triggered in the SIEM solution for the organization from events monitored in the Microsoft 365 Audit that is associated with communication compliance alerts. Investigators are notified of the alert in the SIEM solutions and then they investigate and remediate the alert in the communication compliance solution.
```

ObjectState: Unchanged
```

> **Example of an Audit log entry for a policy with custom keyword match (custom sensitive information type):**

Reviewer: ```suggestion
**Example of a Microsoft 365 Audit log entry for a policy with custom keyword match (custom sensitive information type):**
```

```

> >[!NOTE]

Reviewer: ```suggestion
> [!NOTE]
```

> >Currently, there may be up to a 24h delay between the time a policy match is recorded in the Microsoft 365 Audit and the time in which you can investigate policy matches in communication compliance.

Reviewer: ```suggestion
> Currently, there may be up to a 24-hour delay between the time a policy match is recorded in the Microsoft 365 Audit and the time in which you can investigate policy matches in communication compliance.
```

    | where OfficeWorkload == "Exchange" and Operation == "SupervisionRuleMatch" and UserId == "User1@Contoso.com"
    | sort by TimeGenerated*

> For more information about the Audit logs for Office 365 collected by Azure Sentinel, see [Azure Monitor Logs reference](/azure/azure-monitor/reference/tables/OfficeActivity).

Reviewer: ```suggestion
For more information about the Microsoft 365 Audit logs for Office 365 collected by Azure Sentinel, see [Azure Monitor Logs reference](/azure/azure-monitor/reference/tables/OfficeActivity).
```
